tests/integration/test_pip.py

```diff
@@ -66,6 +66,20 @@
             ),
             id="pip_local_path",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachito-pip-without-metadata.git",
```

Comment:
Should we add a simple Readme to source repository and explain there what's missing and how Cachi2 should react?

Reply:
Okay, why not

Reply:
thx looks great